phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -2280,4 +2281,7 @@ public function format_whitespace($whitespace, $elementStack, $currentDepth) {
         return $whitespace;
     }
 
+    public function format_caption($open, $name, $attrs, $props) {
+        return $open ? '<div class="caption">' : '</div>';
```

Comment:
Isn't the `<figcaption>` tag more appropriate?` although this probably requires more change to support images being put in a figure tag?

Reply:
A `<figcaption>` is a caption for a `<figure>` element but this `<caption>` is used in a `<mediaobject>` element. There is a DocBook example of a `<caption>` [here](https://tdg.docbook.org/tdg/5.2/informalfigure) the HTML of which is also using `<div>`s for `<mediaobject>` and its `<caption>`, so I just followed their lead on this. :-)

Also, I think `<figcaption>` is an HTML element and not a DocBook one.